src/formatDate.js

```diff
@@ -8,7 +9,41 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separador = fromFormat[3];
+  const separadorTo = toFormat[3];
+
+  const newDate = date.split(separador);
```

Comment:
The variable `newDate` is a bit misleading as it does not represent a date but rather an array of date parts. A more descriptive name such as `dateParts` would better represent the contents of the variable.